### PR TITLE
Added example of Animatable use case for custom Canvas animation

### PR DESCRIPTION
**Pull request by riggaroo**
Add Bouncy Loader sample showcasing how to use Animatable for sequential animations. 

https://twitter.com/riggaroo/status/1554430412142903297

### CODE REVIEW

File: MotionCompose/app/src/main/java/com/example/android/compose/motion/MainActivity.kt [hunk 1]
```diff
@@ -17,6 +17,7 @@
 package com.example.android.compose.motion
 
 import android.os.Bundle
+import android.util.Log
```
Comment (IanGClifton) [inline MotionCompose/app/src/main/java/com/example/android/compose/motion/MainActivity.kt:20]: Not necessary anymore?

**Review by IanGClifton — COMMENTED** [review]
I'm not the expert but it looks good to me 👍  I'll let someone else who knows more review for approval though

File: MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/custom/CustomCanvasAnimation.kt [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    val size = remember {
+        mutableStateOf(IntSize.Zero)
+    }
+    Box(
+        Modifier.fillMaxSize()
+            .onSizeChanged {
+                // We get the size change of the whole composable, and use this to determine how
+                // big the ball should be.
+                size.value = it
+            }) {
```
Comment (bentrengrove) [inline MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/custom/CustomCanvasAnimation.kt:71]: Weird formatting

**Review by bentrengrove — APPROVED** [review]
LGTM! I like the graphicsLayer modifier and comment 👍

**Pull request merged**